src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1337,4 +1337,16 @@ Please specify one of them using the `--source` option to proceed.</value>
   <data name="UnableToPurgeInstallDirectory" xml:space="preserve">
     <value>Cannot purge install directory, as it was not created by WinGet</value>
   </data>
+  <data name="BothDisplayAndSuppressNotesFlagsProvided" xml:space="preserve">
+    <value>Both `display-notes` and `suppress-notes` arguments are provided</value>

```

Comment:
Add Comment so that the args are not localized, like `{Locked="display-notes"}` (not sure how to lock two things)

Reply:
Removed localized string as its no longer needed.